package.json

```diff
@@ -28,13 +28,14 @@
   ],
   "scripts": {
     "test": "c8 twist tests/*.js",
-    "build": "npx rollup --format cjs --input index.js --file index.cjs",
+    "build": "npx rollup --external tty --format cjs --input index.js --file index.cjs",
     "deploy": "npm test && git commit --all --message $tag && git tag --sign $tag --message $tag && git push && git push --tags",
     "release": "tag=$npm_package_version npm run deploy && npm publish --access public",
     "prepare": "npm run build"
   },
   "devDependencies": {
     "c8": "*",
+    "rollup": "^2.79.1",
```

Comment:
We don't need to list rollup if we're using npx, do we?

Reply:
npx pulls the package from the registry if it is not found locally.
We can run it directly for smaller package download sizes.

I'm pushing another change.